packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -25,14 +30,17 @@ describe('Payment', function () {
     paymentTx = {
       TransactionType: 'Payment',
       Account: senderWallet.classicAddress,
-      Amount: AMOUNT,
+      Amount: amount,
       Destination: 'rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy',
     }
   })
 
   beforeAll(async () => {
     testContext = await setupClient(serverUrl)
     senderWallet = await generateFundedWallet(testContext.client)
+    // Make sure the amount sent satisfies minimum reserve requirement to fund an account.
+    amount =
+      (await fetchAccountReserveFee(testContext.client)) ?? DEFAULT_AMOUNT
```

Comment:
I agree with this suggestion. Moreover, how can you choose an appropriate `DEFAULT_AMOUNT` ?
Since the XRPL mainnet can vote for arbitrary fee values, I feel we should not advocate for any "Default Fees". If we are unable to retrieve the fees from the latest validated ledger, we should throw an error